### ISSUE TITLE
Cannot open albums after updating to NC 25.0.2

### ISSUE THREAD

**Issue by darguez** [pulsejet/memories#277]
When I try to open Albums a message shows up telling that the request failed with error 500.

![image](https://user-images.githubusercontent.com/96727000/206581970-4e230b12-3afe-4ba4-88a4-c8b7b74854e5.png)

**Comment by pulsejet** [comment]
Memories v4.9.0+?

**Comment by darguez** [comment]
The issue was with Memories v4.8.2, I've just updated to v4.9.1 and the issue seems to be fixed. 👍 There is still an issue with the sidebar that is not correctly shown when you open an album photo, but maybe it is not related to Memories (I will create a new issue).

**Comment by pulsejet** [comment]
Yup, that's a separate bug (it's a Nextcloud bug, actually, but I'll suggest keeping it for the record nonetheless).
